tests/testthat/test-lancelot-vaccination.R

```diff
@@ -3005,68 +3005,86 @@ test_that("run sensible vaccination schedule with waning and boosters", {
+               "There are 2 vaccine classes so 'vacc_skip_progression_rate'")
 
   expect_error(lancelot_parameters(1, "london", rel_susceptibility = c(1, 1, 1),
-                                   vacc_skip_progression_rate = rep(1, 19),
-                                   vacc_skip_from = 2L, vacc_skip_to = 2L),
-               "Require vacc_skip_progression_rate = 0 as vacc_skip_to =")
+                                   vacc_skip_to = c(4, 0, 0)),
+               "There are 3 vaccine classes so the values in 'vacc_skip_to'")
 
   expect_error(lancelot_parameters(1, "london", rel_susceptibility = c(1, 1, 1),
-                                   vacc_skip_progression_rate = rep(0, 19),
-                                   vacc_skip_weight = 1,
-                                   vacc_skip_from = 2L, vacc_skip_to = 2L),
-               "Require vacc_skip_weight = 0 as vacc_skip_to =")
+                                   vacc_skip_to = c(3, 0, 0),
+                                   vacc_skip_weight = c(1, 1, 0)),
+               "Require 0 values in 'vacc_skip_weight' for vaccine strata that")
 
   expect_error(lancelot_parameters(1, "london", rel_susceptibility = c(1, 1, 1),
-                                   vacc_skip_progression_rate = c(1, 1),
-                                   vacc_skip_weight = 1,
-                                   vacc_skip_from = 1L, vacc_skip_to = 3L),
-               "'vacc_skip_progression_rate' must be a scalar or a vector")
+                                   vacc_skip_to = c(3, 0, 0),
+                                   vacc_skip_progression_rate = c(1, 1, 0)),
+               "Require 0 values in 'vacc_skip_progression_rate' for vaccine")
 
   expect_error(lancelot_parameters(1, "london", rel_susceptibility = c(1, 1, 1),
-                                   vacc_skip_from = 1L, vacc_skip_to = 3L,
-                                   vacc_skip_weight = 1.4),
-               "'vacc_skip_weight' must lie in")
+                                   vacc_skip_to = c(2, 0, 0)),
+               "Require vacc_skip_to")
+
+  expect_error(lancelot_parameters(1, "london",
+                                   rel_susceptibility = c(1, 1, 1, 1),
+                                   vacc_skip_to = c(4, 4, 0, 0)),
+               "Cannot have more than one vaccine skip move to the same")
 
   expect_error(lancelot_parameters(1, "london", rel_susceptibility = c(1, 1, 1),
-                                   vacc_skip_from = 1L, vacc_skip_to = 3L,
-                                   vacc_skip_weight = -1),
+                                   vacc_skip_to = c(3, 0, 0),
+                                   vacc_skip_weight = c(1.5, 0, 0)),
                "'vacc_skip_weight' must lie in")
+
+  expect_error(lancelot_parameters(1, "london",
+                                   rel_susceptibility = c(1, 1, 1, 1),
+                                   vacc_skip_to = c(3, 4, 0, 0)),
+               "Cannot have overlapping vaccine skip moves")
```

Comment:
hmm so maybe I didn't understand the overlapping issue in the end. Would be good to clarify what this means